docs/orchestration/faq/upgrading_1.0.md

```diff
@@ -0,0 +1,149 @@
+| -p, --path         | The path to a file containing the flow to run. |
+| -m, --module       | The python module name containing the flow to run. |
+| -n, --name         | The name of a flow to run from the specified file/module/project. If the source contains multiple flows, this must be provided. |
+| --label            | A label to add to the flow run. May be passed multiple times to specify multiple labels. If not passed, the labels from the flow group will be used. |
+| --run-name         | A name to assign to the flow run. |
+| --con              | A key, value pair (key=value) specifying a flow con variable. The value will be interpreted as JSON. May be passed multiple times to specify multiple con values. Nested values may be set by passing a dict. |
+| --param            | A key, value pair (key=value) specifying a flow parameter. The value will be interpreted as JSON. May be passed multiple times to specify multiple parameter values. |
+| --log-level        | The log level to set for the flow run. If passed, the level must be a valid Python logging level name. If this option is not passed, the default level for the flow will be used. Valid values include DEBUG, INFO, WARNING, ERROR, or CRITICAL. |
+| --param-file       | The path to a JSON file containing parameter keys and values. Any parameters passed with `--param` will take precedence over these values. |
+| <span class="no-wrap" style="white-space:nowrap;">--idempotency-key</span>  | A key to prevent duplicate flow runs. If a flow run has already been started with the provided value, the command will display information for the existing run. If using `--execute`, duplicate flow runs will exit with an error. If not using the backing API, this flag has no effect. |
+| --execute          | Execute the flow run in-process without an agent. If this process exits, the flow run will be marked as 'Failed'. |
+| -s, --schedule     | Execute the flow run according to the schedule attached to the flow. If this flag is set, this command will wait between scheduled flow runs. If the flow has no schedule, this flag will be ignored. If used with a non-local run, an exception will be thrown. |
+| -q, --quiet        | Disable verbose messaging about the flow run and just print the flow run ID. |
+| --no-logs          | Disable streaming logs from the flow run to this terminal. Only state changes will be displayed. Only applicable when `--watch` is set. |
+| -w, --watch        | Wait for the flow run to finish executing and display status information. |
+
+For example, run a flow in a script locally:
+
+```bash
+$ prefect run -p hello-world.py
+```
+
+Run a flow with a non-default parameter locally:
+
+```bash
+$ prefect run -m prefect.hello_world --param name=Marvin
+```
+
+Run a registered flow with the backend with custom labels:
+
+```bash
+$ prefect run -n "hello-world" --label example --label hello
+```
+
+Run a registered flow and execute locally without an agent
```

Comment:
This might be a good place to link the 0.15.0 [blog](https://www.prefect.io/blog/prefect-0-15-0-a-new-flow-run-experience/) that details agentless execution?